api/Document.json

```diff
@@ -4447,11 +4447,9 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreen",
           "support": {
             "chrome": {
-              "alternative_name": "webkitIsFullScreen",
```

Comment:
Actually, Chrome does support fullscreen.

* Unprefixed landed in [Chrome 71](https://www.chromestatus.com/features/6596356319739904). This includes WebView.
*  The prefixed version landed in [Chrome 15](https://www.chromestatus.com/features/5259513871466496) on desktop. That means `chrome_android` should say `"18"` and `webview_android` should say `"≤37"`.

If you looked for fullscreen in codesearch and couldn't find it, that's because [it's here](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/fullscreen/document_fullscreen.idl).

Reply:
I failed to notice that I was talking to an experienced contributor. I wouldn't have been so specific in bullet two, otherwise. I apologize for the implied insult.

Reply:
Not a problem, no need to worry!  Thanks for the review, I'll get these version numbers added ASAP.  :wink: